feature_creation_scripts/arctic_ocean_regions/arctic_ocean_regions.py

```diff
@@ -0,0 +1,213 @@
+
+    poly = shapely.geometry.Polygon([(i[0], i[1]) for i in zip(x, y)])
+
+    epsilon = 1e-14
+    minY = numpy.amin(y)
+    wedge = shapely.geometry.Polygon([(epsilon, minY),
+                                      (epsilon**2, -epsilon),
+                                      (0, epsilon),
+                                      (-epsilon**2, -epsilon),
+                                      (-epsilon, minY),
+                                      (epsilon, minY)])
+
+    difference = poly.difference(wedge)
+
+    difference = shapely.ops.transform(stereo_to_lon_lat, difference)
+
+    fc = FeatureCollection()
+
+    geometry = shapely.geometry.mapping(difference)
+    # get rid of the wedge again by rounding the coordinates
+    geometry['coordinates'] = _round_coords(geometry['coordinates'])
+
+    fc.add_feature(
+        {"type": "Feature",
+         "properties": {"name": "Contour {}".format(contourValue),
+                        "author": author,
+                        "object": 'region',
+                        "component": 'ocean'},
+         "geometry": geometry})
+
+    return fc
+
+
+def make_rectangle(lon0, lon1, lat0, lat1, name, author, tags):
+    fc = FeatureCollection()
+
+    fc.add_feature(
+        {"type": "Feature",
+         "properties": {"name": name,
+                        "author": author,
+                        "object": 'region',
+                        "component": 'ocean',
+                        "tags": tags},
+         "geometry": {
+             "type": "Polygon",
+             "coordinates": [[[lon0, lat0],
+                              [lon1, lat0],
+                              [lon1, lat1],
+                              [lon0, lat1],
+                              [lon0, lat0]]]}})
+    return fc
+
+
+def split_rectangle(lon0, lon1, lat0, lat1, name, author, tags, fcContour):
+    fc = make_rectangle(lon0, lon1, lat0, lat1, name, author, tags)
+
+    fcDeep = fc.difference(fcContour)
+
+    props = fcDeep.features[0]['properties']
+    props['name'] = props['name'] + ' Deep'
+    props['tags'] = props['tags'] + ';Deep'
+    props['zmin'] = -1000.
+    props['zmax'] = -400.
+
+    fcShelf = fc.difference(fcDeep)
+
+    props = fcShelf.features[0]['properties']
+    props['name'] = props['name'] + ' Shelf'
+    props['tags'] = props['tags'] + ';Shelf'
+    props['zmin'] = -1000.
+    props['zmax'] = -200.
+
+    fc.merge(fcDeep)
+    fc.merge(fcShelf)
+
+    return fc
+
+
+def main():
+    author = 'Milena Veneziani'
+    arcticTags = 'Arctic_Ocean;Arctic;Arctic_Basin'
+    proshTags = 'Arctic;Proshutinsky'
+
+    # make a geometric features object that knows about the geometric data
+    # cache up a couple of directories
+    gf = GeometricFeatures('../../geometric_data')
+
+    fc = FeatureCollection()
+
+    # Combine old Barentsz_Sea and White_Sea into new Barents Sea 
+    # feature (Barentsz_Sea and White_Sea features will be removed)
```

Comment:
this defines a new Barents Sea as a combination of Barentsz (?) and White Sea. I'd remove both of these features once this PR in..